pkgs/development/python-modules/paste/default.nix

```diff
@@ -26,7 +26,9 @@ buildPythonPackage rec {
     patchShebangs tests/cgiapp_data/
   '';
 
-  propagatedBuildInputs = [
+  build-system = [ setuptools ];
+
+  dependencies = [
     setuptools
```

Comment:
Are you sure setuptools is required at runtime?

Reply:
Thank you for the review!
Yes, `setuptools` is required for `pkg_resources`, see https://github.com/pasteorg/paste/blob/bc1753b00c5f342d28b1af73cd4f662b206c1f37/setup.py#L58